crates/optimism/node/Cargo.toml

```diff
@@ -98,6 +98,7 @@ optimism = [
     "reth-node-core/optimism",
     "reth-optimism-primitives/optimism",
     "reth-optimism-txpool/optimism",
+    "reth-optimism-node/optimism",
```

Comment:
this looks weird, this is the current crate?

why do we need this?

Reply:
oh yeah good catch, thx